setup.py

```diff
@@ -43,6 +43,8 @@
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```

Comment:
assuming 3.10 checks are good we can also add that here

Reply:
Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.

Reply:
Probably should have kept 3.9, but i don't think it will hurt anything to remove it